app/assets/javascripts/corporationtaxfrontend.js

```diff
@@ -0,0 +1,136 @@
+$(document).ready(function() {
```

Comment:
This asset is still called corporation tax frontend
This is really annoying to change you essentially need to go through all of the code for any instance of corporation tax frontend and change it to help and contact.
This may cause issues with scass so make sure the assets are being brought in correctly

Reply:
File renamed.